lib/portfinder.js

```diff
@@ -11,7 +11,7 @@ var fs = require('fs'),
     os = require('os'),
     net = require('net'),
     path = require('path'),
-    async = require('async'),
```

Comment:
after consideration, if we do merge - change the word `async` to `_async` or some other non reserved keyword from ecmascript

/globally

Reply:
Changed to _async. I am not sure this is really needed.